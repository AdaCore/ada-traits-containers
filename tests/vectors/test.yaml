description: 'Basic test for vectors'
driver: 'build_and_exec'
