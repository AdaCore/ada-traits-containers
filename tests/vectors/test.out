one-element vector, element loop => 1
one-element vector, cursor loop => 1
vector, element loop => 1
vector, element loop => 2
vector, element loop => 3
vector, element loop => 4
vector, element loop => 5
vector, element loop => 6
vector, element loop => 7
vector, element loop => 8
vector, element loop => 9
vector, element loop => 10
vector, cursor loop => 1
vector, cursor loop => 2
vector, cursor loop => 3
vector, cursor loop => 4
vector, cursor loop => 5
vector, cursor loop => 6
vector, cursor loop => 7
vector, cursor loop => 8
vector, cursor loop => 9
vector, cursor loop => 10
vector after delete, element loop => 2
vector after delete, element loop => 3
vector after delete, element loop => 5
vector after delete, element loop => 7
vector after delete, element loop => 8
vector after delete, element loop => 9
vector after swap, element loop => 9
vector after swap, element loop => 3
vector after swap, element loop => 5
vector after swap, element loop => 7
vector after swap, element loop => 8
vector after swap, element loop => 2
vector after swap same element, element loop => 9
vector after swap same element, element loop => 3
vector after swap same element, element loop => 5
vector after swap same element, element loop => 7
vector after swap same element, element loop => 8
vector after swap same element, element loop => 2
vector after replace, element loop => 100
vector after replace, element loop => 3
vector after replace, element loop => 5
vector after replace, element loop => 7
vector after replace, element loop => 8
vector after replace, element loop => 600
